### ISSUE TITLE
[a11y] - search and highlight ui

### ISSUE THREAD

**Issue by transkamp** [open-pioneer/trails-openlayers-base-packages#446]
### Issue description

open https://open-pioneer.github.io/trails-demo/openlayers-base-packages/dev/samples/showcase/?demo=searchAndHighlight

- add tooltip to "Suchen leeren" button (red)
- add aria-hidden="true" Attribute to close icon and magnifier icon otherwhise the a11y parser detects an error because the svg icon has no alt attribute (magenta)

![Image](https://github.com/user-attachments/assets/b0b3ae23-0e0b-4f92-bb56-c337a53b6c7a)

### Validations

- [x] Read and follow our [code of conduct](https://github.com/open-pioneer/.github/blob/main/CODE_OF_CONDUCT.md)
- [x] Read the [contributing guidelines](https://github.com/open-pioneer/trails-starter/blob/main/docs/Contributing.md)
- [x] Check that there isn't already a similar issue
